reducer/src/main/java/com/graphicsfuzz/reducer/reductionopportunities/GlobalVariableDeclToExprReductionOpportunity.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 The GraphicsFuzz Project Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.graphicsfuzz.reducer.reductionopportunities;
+
+import com.graphicsfuzz.common.ast.visitors.VisitationDepth;
+
+public class GlobalVariableDeclToExprReductionOpportunity extends AbstractReductionOpportunity {
+  GlobalVariableDeclToExprReductionOpportunity(VisitationDepth depth) {
+    super(depth);
+  }
+
+  @Override
+  void applyReductionImpl() {
+  }
+
+  @Override
+  public boolean preconditionHolds() {
```

Comment:
Similarly, I would throw an exception here to indicate that it's not implemented.

This reduces the risk that something gets executed despite not being ready.